platforms/qnx/aarch64-qnx.toolchain.cmake

```diff
@@ -30,7 +30,7 @@ set(CMAKE_MODULE_LINKER_FLAGS "${LINKER_FLAGS} ${CMAKE_MODULE_LINKER_FLAGS}")
 set(CMAKE_EXE_LINKER_FLAGS "${LINKER_FLAGS} ${CMAKE_EXE_LINKER_FLAGS}")
 
 if(NOT DEFINED CMAKE_LINKER)
```

Comment:
Looking at this the whole check starting with:
```
if(NOT DEFINED CMAKE_LINKER)
```
can be removed, as `CMAKE_LINKER` is always defined. Please remove it.

It should contain a proper path thanks to that typo fix.